.drone.yml

```diff
@@ -76,8 +76,10 @@ steps:
       from_secret: docker_username
     password:
       from_secret: docker_password
-    spec: manifest.tmpl
-    build.tag: "${DRONE_TAG/+/-}"
+    target: rancher/system-agent-installer-rke2:${DRONE_TAG/+/-}
+    template: rancher/system-agent-installer-rke2:${DRONE_TAG/+/-}-OS-ARCH
```

Comment:
Would this need to use `${GOOS}-${GOARCH}` (or similar)? Or, are the `OS` and `ARCH` values automatically interpreted as vars in this plugin?

Reply:
Per the docs that is the correct format for the template: http://plugins.drone.io/drone-plugins/drone-manifest/